recipe/github_com__sodre__yq__entry_points.patch

```diff
@@ -0,0 +1,54 @@
+diff --git a/scripts/xq b/scripts/xq
+deleted file mode 100755
+index 3435723..0000000
+--- a/scripts/xq
++++ /dev/null
+@@ -1,5 +0,0 @@
+-#!/usr/bin/env python
+-
+-from yq import main
+-
+-main(input_format="xml")
```

Comment:
remove scripts/xq. Doing this in the patch guarantees that if upstream changes the way "xq" is called that we can pick it up as a build failure and redo the patch.